src/main/java/com/dreammaster/gthandler/GT_Loader_Wires.java

```diff
@@ -16,29 +16,30 @@ public void run() {
-        makeWires(Materials.VibrantAlloy, 11530, 2L, 4L, 4L, gregtech.api.enums.GTValues.V[4], true, false);
-        makeWires(Materials.MelodicAlloy, 11550, 2L, 4L, 4L, gregtech.api.enums.GTValues.V[5], true, false);
-        makeWires(Materials.StellarAlloy, 11570, 4L, 8L, 6L, gregtech.api.enums.GTValues.V[6], true, false);
-        makeWires(Materials.Trinium, 11450, 4L, 8L, 6L, GTValues.V[7], true, false);
-        makeWires(Materials.ElectrumFlux, 1900, 1L, 2L, 3L, GTValues.V[8], true, false);
-        makeWires(Materials.Bedrockium, 11310, 1L, 32L, 2L, GTValues.V[9], true, false);
-        makeWires(Materials.Osmiridium, 11610, 1L, 2L, 8L, GTValues.V[6], true, false);
-        makeWires(Materials.HSSE, 11590, 2L, 4L, 6L, GTValues.V[6], true, false);
-        makeWires(Materials.HSSS, 11470, 4L, 8L, 6L, GTValues.V[9], true, false);
-        makeWires(Materials.Draconium, 11330, 4L, 32L, 8L, GTValues.V[10], true, false);
-        makeWires(Materials.NetherStar, 11350, 4L, 16L, 4L, GTValues.V[11], true, false);
-        makeWires(Materials.Quantium, 11370, 4L, 16L, 2L, GTValues.V[12], true, false);
-        makeWires(Materials.BlackPlutonium, 11390, 4L, 16L, 1L, GTValues.V[13], false, false);
-        makeWires(Materials.DraconiumAwakened, 11410, 4L, 16L, 1L, GTValues.V[14], false, false);
+        makeWires(Materials.ElectricalSteel, 11490, 1L, 2L, 3L, GTValues.V[2], true, false);
+        makeWires(Materials.EnergeticAlloy, 11510, 1L, 2L, 4L, gregtech.api.enums.GTValues.V[3], true, false);
+        makeWires(Materials.VibrantAlloy, 11530, 3L, 6L, 6L, gregtech.api.enums.GTValues.V[4], true, false);
+        makeWires(Materials.EndSteel, 1860, 3L, 6L, 6L, gregtech.api.enums.GTValues.V[5], true, false);
+        makeWires(Materials.MelodicAlloy, 11550, 2L, 4L, 6L, gregtech.api.enums.GTValues.V[6], true, false);
+        makeWires(Materials.StellarAlloy, 11570, 12L, 24L, 12L, gregtech.api.enums.GTValues.V[8], true, false);
+        makeWires(Materials.Trinium, 11450, 3L, 6L, 6L, GTValues.V[7], true, false);
+        makeWires(Materials.ElectrumFlux, 1900, 3L, 2L, 8L, GTValues.V[8], true, false);
+        makeWires(Materials.Bedrockium, 11310, 6L, 48L, 3L, GTValues.V[9], true, false);
+        makeWires(Materials.Osmiridium, 11610, 1L, 2L, 16L, GTValues.V[7], true, false);
+        makeWires(Materials.HSSE, 11590, 4L, 8L, 6L, GTValues.V[6], true, false);
+        makeWires(Materials.HSSS, 11470, 4L, 8L, 8L, GTValues.V[9], true, false);
+        makeWires(Materials.Draconium, 11330, 32L, 96L, 12L, GTValues.V[10], true, false);
+        makeWires(Materials.NetherStar, 11350, 64L, 160L, 4L, GTValues.V[11], true, false);
+        makeWires(Materials.Quantium, 11370, 32L, 128L, 4L, GTValues.V[12], true, false);
+        makeWires(Materials.BlackPlutonium, 11390, 8L, 8L, 8L, GTValues.V[13], false, false);
+        makeWires(Materials.DraconiumAwakened, 11410, 64L, 64L, 24L, GTValues.V[14], false, false);
```

Comment:
I think buffing awakened draconium this much will turn it into the best option between draconium and infinity, which I'm not sure is a positive change. Currently draconium starts to feel slightly strained before the time when infinity is basically no cost to make wires out of, and this change I think would smooth that gap out which I think is not a good thing

Reply:
I see what you mena, but at the same time, it deserves _some_ kind of buff, it's beyond shit for anything. Plus, the superconductors are a very viable option at this point, this just gives another option.

Reply:
I just am not sure that there is room for it to be a good cable without being too meta-disruptive. At some point cables simplify a lot (draconium -> infinity) and I think since draconium can't really be nerfed without breaking too many bases, we should at least keep the wider gap between it and the final wire.

Ultimately not every wire has to be viable, nor is there room for them all to be, some can just be used for crafting

Reply:
Inifnity is availeable at scale in early UEv, late UHV if you stretch it. draconium will stay the most used regardless, likely due to the patterns being made and people disinclined to remake them, with awdrac only filling in for barely half a tier on more amp-dense setups that could be substituted by a few ingots of infinity regardless. Plus, awdrac is on a t8 planet as a small ore, so you need to be at least halfway through uvh to obtain.

Reply:
I still just disagree on making another extremely powerful wire material between draconium and infinity, I think the dynamic these two wires have is fine and doesn't need a new alternative

Reply:
Alright. 8a? Same amps, but the larger loss should keep drac in use.

Reply:
Sure